bridge/client/app/project-board/project-board.component.ts

```diff
@@ -22,32 +22,43 @@ export class ProjectBoardComponent implements OnInit, OnDestroy {
   public error$: Observable<string | undefined> = this._errorSubject.asObservable();
   public isCreateMode$: Observable<boolean>;
   public hasUnreadLogs$: Observable<boolean>;
+  private readonly uniformLogPollingInterval = 2 * 60_000;
```

Comment:
Does this need to be a field? It seems unnecessary to expose this variable to the whole component. Could this also be a constant outside of the class (`defaultUniformLogPollingInterval`), that is being used if `initialDelayMillis` is greater than 0? Something like this:

```
const defaultUniformLogPollingInterval = 2 * 60_000;

...
export class ...
...
    const uniformLogPollingInterval = initialDelayMillis > 0 ? defaultUniformLogPollingInterval : 0;
    const uniformLogTimer$ = projectName$.pipe(
      switchMap(() => AppUtils.createTimer(0, uniformLogPollingInterval)),
      takeUntil(this.unsubscribe$)
    );
```